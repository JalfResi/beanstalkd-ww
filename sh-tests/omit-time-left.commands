put 0 0 5 1
a
stats-job 1
quit
