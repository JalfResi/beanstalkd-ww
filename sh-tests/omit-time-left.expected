INSERTED 1
OK 135
---
id: 1
tube: default
state: ready
pri: 0
age: 0
delay: 0
ttr: 5
time-left: 0
reserves: 0
timeouts: 0
releases: 0
buries: 0
kicks: 0

